### ISSUE TITLE
Incorrectly says to end paragraph with a full stop in markdown

### ISSUE THREAD

**Issue by rubenwardy** [wysiib/linter-languagetool#31]
```
Suggests a full stop here ------------------------|
                                                  v
This is an example sentence in Markdown. Sentences
after a full stop which then flow on to a new line
results in a fullstop being wrongly suggested.

This does not happen when you flow onto a new line
without a fullstop on the line.
```

Not sure whether it's possible to work around this here. According to a LanguageTool maintainer, plugins like this should fix this issue:

> **danielnaber**
>
> I consider this out of scope for LT. Instead, editors should integrate LT and they should do the parsing of the text (markup vs. text, hand only the text to LT).
> 
> https://github.com/languagetool-org/languagetool/issues/710

This is similar to #7 except this is about whitespace rather than about ignoring markup

**Comment by wysiib** [comment]
Indeed, this is very closely related to #7, where @hesstobi suggested a way to proceed, which will be integrated in upcoming versions of language tool. This issue (and related ones) follow us since quite some time. However, it is complicated to decide on the best way to proceed (e.g. a way that solves the problem for several markup languages and avoids special casing as much as possible).
